charts/azuremonitor-containers/templates/omsagent-arc-k8s-crd.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or ( contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower) ) ( contains "microsoft.kubernetes/connectedclusters" (.Values.omsagent.env.clusterId | lower)) }}
+{{- if or ( contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower)) ( contains "microsoft.kubernetes/connectedclusters" (.Values.omsagent.env.clusterId | lower)) ( contains "microsoft.hybridcontainerservice/provisionedclusters" (.Values.Azure.Cluster.ResourceId | lower)) ( contains "microsoft.hybridcontainerservice/provisionedclusters" (.Values.omsagent.env.clusterId | lower)) }}
```

Comment:
This if clause can be simplified with just checking ".Values.Azure.Cluster.ResourceId" and we can remove the conditions related to ".Values.omsagent.env.clusterId"  which is not required. This was added in the begining when we dont have extension model.

Reply:
changed to only check resourceId